### PR TITLE
Add billing section under Storage

### PR DESCRIPTION
**Pull request by aayushmahapatra**

### CODE REVIEW

File: pages/storage/billing.mdx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Billing
+
+Our billing plans offer flexibility to cater to various project needs.
+Below is an overview of our pricing plans, each tailored to specific requirements.
+
+- **STARTER:** Ideal for personal or non-commercial projects. This plan offers 5 GB of total storage and 10 GB of bandwidth, giving you a strong start with no monthly cost.
+- **LITE:** Empowering projects and developers with extensive authority over their IPFS gateways. This plan includes 40 GB of total storage and 100 GB of bandwidth.
+- **PRO:** Addressing data and infrastructure demands of marketplaces, apps, and games with substantial requirements. Enjoy 150 GB of total storage and 400 GB of bandwidth.
+- **WAGMI:** Designed to serve apps and projects that are scaling exponentially. This plan provides 800 GB of total storage and a generous 2.5 TB of bandwidth.
+- **ENTERPRISE:** Seeking even greater possibilities? Envisioning a complex integration? Share detailed information about your company and project in the form below, and we will promptly respond to your inquiry with a custom plan tailored to your unique needs.
```
Comment (rekpero) [inline pages/storage/billing.mdx:10]: add the end of the para, just mention connect with the team for more details and add the contact link

File: pages/storage/billing.mdx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+| **Plan**           | **Protocol** | **Price/month** | **Total Storage** | **Bandwidth** |
+| ------------------ | ------------ | --------------- | ----------------- | ------------- |
+| SPHERON STARTER    | IPFS         | FREE            | 5 GB              | 10 GB         |
+| SPHERON LITE       | IPFS         | $3              | 40 GB             | 100 GB        |
+| SPHERON PRO        | IPFS         | $10             | 150 GB            | 400 GB        |
+| SPHERON WAGMI      | IPFS         | $50             | 800 GB            | 2.5 TB        |
+| SPHERON ENTERPRISE | IPFS         | Contact Sales   | Custom            | Custom        |
```
Comment (rekpero) [inline pages/storage/billing.mdx:18]: in contact sale, mention the link

File: pages/storage/billing.mdx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## Flexible Plan Upgrades
```
Comment (rekpero) [inline pages/storage/billing.mdx:20]: mention bonuses that users can add by themself and their pricing

**Review by rekpero — CHANGES_REQUESTED** [review]

File: pages/storage/billing.mdx (hunk 3 above)
Comment (aayushmahapatra) [reply pages/storage/billing.mdx:20]: Done

File: pages/storage/billing.mdx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+| SPHERON PRO        | IPFS         | $10                                                                        | 150 GB            | 400 GB        |
+| SPHERON WAGMI      | IPFS         | $50                                                                        | 800 GB            | 2.5 TB        |
+| SPHERON ENTERPRISE | IPFS         | [Contact Sales](https://meetings.hubspot.com/prashant-maurya/scaling-demo) | Custom            | Custom        |
+
+## Flexible Plan Upgrades
+
+- You have the flexibility to upgrade to bigger plans as your project grows.
+- However, if you wish to switch to a smaller plan, please note that you will need to wait until your current plan expires.
+
+Explore our range of plans:
```
Comment (rekpero) [inline pages/storage/billing.mdx:25]: these are Ad-ons, not plan. Please mention it properly

**Review by rekpero — CHANGES_REQUESTED** [review]

File: pages/storage/billing.mdx (hunk 4 above)
Comment (rekpero) [reply pages/storage/billing.mdx:25]: also mention the pricing for each

File: pages/storage/billing.mdx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+| **Ad-ons**         | **Price**        |
+| ------------------ | ---------------- |
+| Parallel Uploads   | $12 / upload     |
+| Arweave Storage    | $10 / 1.5 GB     |
+| IPFS Storage       | $0.3 / 3 GB      |
+| Hns Domains        | $1 / domain      |
+| Ens Domains        | $1 / domain      |
+| CDN Bandwidth      | $40 / 100 GB     |
+| IPFS Bandwidth     | $40 / 100 GB     |
+| Image Optimization | $4 / 1000 images |
```
Comment (rekpero) [inline pages/storage/billing.mdx:36]: remove this

File: pages/storage/billing.mdx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+- You have the flexibility to upgrade to bigger plans as your project grows.
+- However, if you wish to switch to a smaller plan, please note that you will need to wait until your current plan expires.
+
+Explore our range of Ad-ons:
+
+| **Ad-ons**         | **Price**        |
+| ------------------ | ---------------- |
+| Parallel Uploads   | $12 / upload     |
+| Arweave Storage    | $10 / 1.5 GB     |
+| IPFS Storage       | $0.3 / 3 GB      |
```
Comment (rekpero) [inline pages/storage/billing.mdx:31]: remove this

**Review by rekpero — CHANGES_REQUESTED** [review]

File: pages/storage/billing.mdx [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+- However, if you wish to switch to a smaller plan, please note that you will need to wait until your current plan expires.
+
+Explore our range of Ad-ons:
+
+| **Ad-ons**       | **Price**    |
+| ---------------- | ------------ |
+| Parallel Uploads | $12 / upload |
+| Arweave Storage  | $10 / 1.5 GB |
+| Hns Domains      | $1 / domain  |
+| Ens Domains      | $1 / domain  |
```
Comment (rekpero) [inline pages/storage/billing.mdx:32]: HNS and ENS

**Review by rekpero — CHANGES_REQUESTED** [review]

File: pages/storage/billing.mdx [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+Explore our range of Ad-ons:
+
+| **Ad-ons**       | **Price**    |
+| ---------------- | ------------ |
+| Parallel Uploads | $12 / upload |
+| Arweave Storage  | $10 / 1.5 GB |
+| HNS Domains      | $1 / domain  |
+| ENS Domains      | $1 / domain  |
+| CDN Bandwidth    | $40 / 100 GB |
```
Comment (rekpero) [inline pages/storage/billing.mdx:33]: can you add gateway bandwidth? I just remembered it

**Review by rekpero — CHANGES_REQUESTED** [review]

File: pages/storage/billing.mdx (hunk 8 above)
Comment (rekpero) [reply pages/storage/billing.mdx:33]: it's $40/100GB

Comment (aayushmahapatra) [reply pages/storage/billing.mdx:33]: Can you tell me the price for it?

**Review by rekpero — APPROVED** [review]

**Pull request merged**